tests/integrations/nvidia/guardrails_test.py

```
@@ -0,0 +1,53 @@
+from typing import Generator
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)